Error while checking build/O2/o2-dataflow-cs8 for edc2f504650f6bc1180648410f27b52c2f78fb5f at 2022-10-05 19:50:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/10003-slc8_x86-64/0/Detectors/TRD/reconstruction/src/CruRawReader.cxx:197:23: error: 'e' was not declared in this scope
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/10003/edc2f504650f6bc1180648410f27b52c2f78fb5f/build_O2_o2-dataflow-cs8/pretty.html).

--- a/Detectors/TRD/reconstruction/src/CruRawReader.cxx
+++ b/Detectors/TRD/reconstruction/src/CruRawReader.cxx
@@ -155,7 +155,11 @@ int CruRawReader::processHBFs()
   while (!o2::raw::RDHUtils::getStop(rdh)) { // carry on till the end of the event.
     if (mOptions[TRDVerboseBit]) {
       LOG(info) << "Current RDH is as follows:";
-      o2::raw::RDHUtils::printRDH(rdh);
+      try {
+        o2::raw::RDHUtils::printRDH(rdh);
+      } catch (std::runtime_error& e) {
+        LOG(error) << e.what();
+      }
     }
     if (!checkRDH(rdh)) {
       return -1;
@@ -187,7 +191,11 @@ int CruRawReader::processHBFs()
 
     if (mOptions[TRDVerboseBit]) {
       LOG(info) << "Next RDH is as follows:";
-      o2::raw::RDHUtils::printRDH(rdh);
+      try {
+        o2::raw::RDHUtils::printRDH(rdh);
+      } catch (std::runtime_error& e) {
+        LOG(error) << e.what();
+      }
     }
 
     if (!o2::raw::RDHUtils::getStop(rdh) && offsetToNext >= mCurrRdhPtr - mDataBufferPtr) {

